public/apps/configuration/panels/audit-logging/audit-logging-edit-settings.tsx

```diff
@@ -121,9 +133,13 @@ export function AuditLoggingEditSettings(props: AuditLoggingEditSettingProps) {
       };
 
       if (props.setting === 'general') {
-        addSuccessToast('General settings saved');
+        addSuccessToast(
+          `General settings saved ${getClusterInfoIfEnabled(dataSourceEnabled, dataSource)}`
```

Comment:
Would existence of the datasource imply that MDS is enabled?

I'm wondering if the call to `const { dataSource, setDataSource } = useContext(DataSourceContext)!;` can be done within `getClusterInfoIfEnabled`. 

I'd also be in favor of shortening the fn name to `getClusterLabel`.

Reply:
From how I see it, I don't think we can push the context into the helper function, since in addition to showing the UI text/toast surrounding the datasource, we also need to modify the API call which occurs outside of this helper function. 
> Would existence of the datasource imply that MDS is enabled?
I am not sure what you mean by this. There are two checks in the helper function - one that checks whether multi datasource is enabled or not, and if it is we try to pull the label of the selected datasource to show in the UI text/toast. 
> I'd also be in favor of shortening the fn name to getClusterLabel.
Sure I can make that change - When initially naming the function I thought it might be confusing to have a getClusterLabel function if the multidatasource feature is disabled, meaning there is no concept of outside clusters. @scrawfor99 @DarshitChanpura and opinions on this function?

Reply:
Context should not be pushed in the helper function. But I agree that the function can be renamed to `getClusterInfo`

Reply:
Renamed